tests/baselines/reference/decoratorOnClass5.es6.js

```diff
@@ -16,10 +16,10 @@ var __decorate = (this && this.__decorate) || function (decorators, target, key,
     else for (var i = decorators.length - 1; i >= 0; i--) if (d = decorators[i]) r = (c < 3 ? d(r) : c > 3 ? d(target, key, r) : d(target, key)) || r;
     return c > 3 && r && Object.defineProperty(target, key, r), r;
 };
-let C_1 = class C {
+let C_1;

```

Comment:
This is wrong. This was fixed by #8786 in master and indicates a change we also need to make here.